demo/list-box-presentation-demos.html

```diff
@@ -0,0 +1,90 @@
+<dom-module id="list-box-presentation-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Separators</h3>
+    <vaadin-demo-snippet id="list-box-presentation-demos-separators">
+      <template preserve-content>
+        <vaadin-list-box>
+          <vaadin-item disabled>Weekdays</vaadin-item>
+          <hr>
+          <vaadin-item>Monday</vaadin-item>
+          <vaadin-item>Tuesday</vaadin-item>
+          <vaadin-item>Wednesday</vaadin-item>
+          <vaadin-item>Thursday</vaadin-item>
+          <vaadin-item>Friday</vaadin-item>
+          <hr>
+          <vaadin-item>Saturday</vaadin-item>
+          <vaadin-item>Sunday</vaadin-item>
+        </vaadin-list-box>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Customizing listbox item label</h3>
+    <vaadin-demo-snippet id="list-box-presentation-demos-custom-label">
+      <template preserve-content>
+        <b>User</b>
+        <vaadin-list-box>
+          <vaadin-item value="1" selected>
+            <img src="https://randomuser.me/api/portraits/women/43.jpg" />
+            Gabriella
+          </vaadin-item>
+          <vaadin-item value="2">
+            <img src="https://randomuser.me/api/portraits/men/77.jpg" />
+            Rudi
+          </vaadin-item>
+          <vaadin-item value="3">
+            <img src="https://randomuser.me/api/portraits/men/35.jpg" />
+            Hamsa
+          </vaadin-item>
+          <vaadin-item value="3">
```

Comment:
There are two items with `value="3"`, this latter one should be fixed to `value="4"`.

Reply:
Done.